### ISSUE TITLE
duplicate-code warning with concurrent run

### ISSUE THREAD

**Issue by Nikita-Boyarskikh** [PyCQA/pylint-django#196]
`duplicate-code` warnings are not detected when pylint is started with the parameter `jobs` other then 1. If this is the expected behavior, it's worth fixing it in the documentation, I think.

<details>
<summary>pip freeze --all</summary>

```
arrow==0.12.1
astroid==2.0.4
autopep8==1.4
bandit==1.5.1
Django==2.1.1
django-uuslug==1.1.8
djangorestframework==3.8.2
flake8==3.5.0
flake8-per-file-ignores==0.6
gitdb2==2.0.4
GitPython==2.1.11
isort==4.3.4
lazy-object-proxy==1.3.1
mccabe==0.6.1
pathmatch==0.2.1
pbr==4.2.0
pip==18.0
psycopg2==2.7.5
psycopg2-binary==2.7.5
pycodestyle==2.3.1
pyenchant==2.0.0
pyflakes==1.6.0
pylint==2.1.1
pylint-celery==0.3
pylint-django==2.0.2
pylint-plugin-utils==0.4
python-dateutil==2.7.3
python-slugify==1.2.6
pytz==2018.5
PyYAML==3.13
setuptools==40.2.0
six==1.11.0
smmap2==2.0.4
stevedore==1.29.0
typing==3.6.6
Unidecode==1.0.22
wheel==0.31.1
wrapt==1.10.11
```

</details>

**Comment by atodorov** [comment]
`duplicate-code` comes from pylint itself, not pylint_django. Can you reproduce the problem when pylint_django plugin is not loaded ? If yes then open a bug against pylint.

If no it could be that pylint_django somehow screws things up in a concurrent execution and I have no idea how to begin figuring this one out.
